mpy-cross/mpconfigport.h

```diff
@@ -93,11 +93,11 @@
 #ifdef __LP64__
 typedef long mp_int_t; // must be pointer size
 typedef unsigned long mp_uint_t; // must be pointer size
-#elif defined ( __MINGW32__ ) && defined( _WIN64 )
+#elif defined (__MINGW32__) && defined(_WIN64)
```

Comment:
I know it is not related to this issue, but might as well make space after `defined` consistent here and below.

Reply:
Turns out uncrustify has an option just for this, so I've enabled that as well in this PR.